tests/scripts/arp_responder.py

```diff
@@ -125,9 +132,30 @@ def main():
         if vlan is not None:
             ip_sets[iface]['vlan'].append(binascii.unhexlify(vlan_tag))
 
+    for iface in ip_sets:
+        arp_filter_entries = []
+        icmp_filter_entries = []
+        for ip in ip_sets[iface]:
+            ip_address = ipaddress.ip_address(ip)
+            if ip_address.version == 4:
+                arp_filter_entries.append(f'arp[24:4] = 0x{int.from_bytes(ip_address.packed, "big"):0x}')  # noqa: E231
+            else:
+                ipv6_address_integer = int.from_bytes(ip_address.packed, "big")
+                # libpcap on Buster doesn't support looking into ICMPv6 header directly, so look from the IPv6 header
```

Comment:
> libpcap on Buster doesn't support looking into ICMPv6 header directl

How does this limit the cherry picking of this PR?

Reply:
Cherry-picking is not affected. PTF is currently on Buster for all branches, and the BPF filter used below can be used even on newer versions of libpcap.